services/spar/test-integration/Test/Spar/APISpec.hs

```diff
@@ -638,21 +638,97 @@ specCRUDIdentityProvider = do
           (owner, _, (^. idpId) -> idpid) <- registerTestIdP
           callIdpUpdate' (env ^. teSpar) (Just owner) idpid (IdPMetadataValue "<NotSAML>bloo</NotSAML>" undefined)
             `shouldRespondWith` ((== 400) . statusCode)
-    describe "issuer changed to one that already exists in another team" $ do
+    describe "issuer changed to one that already exists in *another* team" $ do
       it "rejects" $ do
         env <- ask
         (owner1, _, (^. idpId) -> idpid1) <- registerTestIdP
         (_, _, _, (IdPMetadataValue _ idpmeta2, _)) <- registerTestIdPWithMeta
         callIdpUpdate' (env ^. teSpar) (Just owner1) idpid1 (IdPMetadataValue (cs $ SAML.encode idpmeta2) undefined)
-          `shouldRespondWith` checkErr (== 400) "idp-used-in-other-team"
-    describe "issuer changed to one that is new" $ do
+          `shouldRespondWith` checkErr (== 400) "idp-issuer-in-use"
```

Comment:
Why the change in error message type?

Reply:
i thought it was an error if the idp issuer is already in use in another team, but that's silly.  doesn't matter where the issuer is already in use, whether other team or not.

Reply:
The former label is not in prod yet, and I haven't communicated it to anybody, so it should be fine.